features/LIBRARY.feature

```diff
@@ -0,0 +1,93 @@
+Feature: LIBRARY
+  Scenario: Editing books in the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the edit button
+    And We change the title and / or author
+    And Confirm with the Ok button
+    Then The author and / or title for this book are changed
+
+  Scenario: Adding books to the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the Add book button
+    And We complete the form
+    And Confirm with the Ok button
+    Then The book was added to the table
+
+  Scenario: Delete books from the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the delete button
+    Then The book has been removed from the table
+
+  Scenario: Sorting alphabetically by title in ascending order
+    Given We are logged in and we are in the LIBRARY section
+    When Click on the drop-down list next to SORT BY and select Title ascending
+    Then The books in the table have been sorted alphabetically by titles in ascending order
+
+  Scenario: Sort alphabetically by title in descending order
+    Given We are logged in and we are in the LIBRARY section
+    When Click on the drop-down list next to SORT BY and select Title descending
+    Then The books in the table have been sorted alphabetically by descending titles
+
+  Scenario: Sort alphabetically by borrowings, descending
+    Given We are logged in and we are in the LIBRARY section
+    When Click on the drop-down list next to SORT BY and select Rented descending
+    Then The books in the table have been sorted descending by rented
+
+  Scenario: Sort alphabetically on loans ascending
+    Given We are logged in and we are in the LIBRARY section
+    When Click on the drop-down list next to SORT BY and select Rented ascending
+    Then the books in the table have been sorted in ascending order by rented
```

Comment:
Po Given When Then jeśli stosujemy duże litery to trzeba się tego zawsze konsekwentnie trzymać i stosować to wszędzie